mobile/lib/features/welcome/welcome_screen.dart

```diff
@@ -36,75 +39,182 @@ class _WelcomeScreenState extends State<WelcomeScreen> {
+                                const SizedBox(
+                                  width: 250,
+                                  child: Text("10101 is still in testing and may contain defects.",
+                                      softWrap: true),
+                                ),
+                                TenTenOneSwitch(
+                                    value: _betaDisclaimer,
+                                    onChanged: (value) => setState(() => _betaDisclaimer = value)),
+                              ],
+                            ),
+                          ),
+                          const SizedBox(height: 10),
+                          Container(
+                            padding: const EdgeInsets.all(15),
+                            decoration: BoxDecoration(
+                                color: tenTenOnePurple.shade300.withOpacity(0.2),
+                                borderRadius: BorderRadius.circular(10.0)),
+                            child: Row(
+                              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                              children: [
+                                const SizedBox(
+                                  width: 250,
+                                  child: Text(
+                                      "If I lose my seed phrase, my funds will be lost forever",
+                                      softWrap: true),
+                                ),
+                                TenTenOneSwitch(
+                                    value: _loseDisclaimer,
+                                    onChanged: (value) => setState(() => _loseDisclaimer = value)),
+                              ],
+                            ),
+                          ),
+                          const SizedBox(height: 10),
+                          Form(
+                            key: _formKey,
+                            child: TextFormField(
+                              keyboardType: TextInputType.emailAddress,
+                              initialValue: _email,
+                              decoration: InputDecoration(
+                                  border:
+                                      OutlineInputBorder(borderRadius: BorderRadius.circular(10.0)),
+                                  enabledBorder: OutlineInputBorder(
+                                      borderRadius: BorderRadius.circular(10.0),
+                                      borderSide: BorderSide(
+                                          color: tenTenOnePurple.shade300.withOpacity(0.2))),
+                                  filled: true,
+                                  fillColor: tenTenOnePurple.shade300.withOpacity(0.2),
+                                  labelText: 'Email (optional)',
+                                  labelStyle: const TextStyle(color: Colors.black87),
+                                  hintText: 'Let us know how to reach you (optional)'),
```

Comment:
🙃 Redundant with the `labelText`.

```suggestion
                                  hintText: 'Let us know how to reach you'),
```